### DISCUSSION TITLE
MinIO server on macOS version 10.13.6 getting timed out for any client request

### DISCUSSION THREAD

**Discussion by fkt-wtag** [minio/minio#14375] [Q&A]
Hi everyone! Hope everyone are doing well!
 I am really struggling with minio server in my local environment where the OS is a **macOS version 10.13.6**. I tried minio server setup using 3 different ways. i.e.

- Downloading latest binary using https://dl.min.io/server/minio/release/darwin-amd64/minio
- Through brew using brew install minio/stable/minio
- Through docker by getting the latest image from bitnami/hub.docker.com

Then for example I am starting the server using `MINIO_ROOT_USER=minioadmin MINIO_ROOT_PASSWORD=minioadmin ./minio server data/ --console-address ":9001"` .

In every case the server is starting properly and I am explicitly defining the console-address . But the problem is, I can't login to console by using any exposed console URL for example, `http://192.168.0.110:9001/` or `http://127.0.0.1:9001/` or `http://localhost:9001/`.

So when I do try to login using my ROOT_USER and ROOT_PASSWORD, after a certain period server responded with a timeout error.

However, I checked my mac's firewall settings as well and found that for all instances of minio either it is native or docker image the incoming connections are allowed. 

My final goal is using minio for gcs gateway and for that also the issue is same since minio server is not even working properly in a non gateway mode.

**Some additional info:** 
I tried working with gcs gateway as well. But problem is, when I tried to get or put any resources to gcs then again minio client hangs up and got timed out after a certain period. On the other hand, through **terminal** and **mc** I can do all types of communication with gcs and buckets. 

Any help would be much appreciated! Thanks in advance

**Comment by riderx** [comment]
Did you find a solution i think i have the same issue
